pkg/server/helper.go

```diff
@@ -28,13 +28,13 @@ func GeneratePTS(depBody deploymentPost, org, env string) (v1.PodTemplateSpec, e
 		return v1.PodTemplateSpec{}, errors.New("No URI set")
 	}
 
-	cdir := os.Getenv("POD_CDIR")
-	if cdir == "" {
-		cdir = "10.1.0.0/16"
+	cidr := os.Getenv("POD_CIDR")
+	if cidr == "" {
+		cidr = "10.1.0.0/16"
 	}
 
 	var tempPaths string
-	var intPort int
+	containerPorts := make([]v1.ContainerPort, len(depBody.Paths))
```

Comment:
That's interesting. The tricky thing is that I need to have the `containerPorts` value at a higher scope, outside of the if/else blocks, since I use it later on line 149.  I